src/grt/src/GlobalRouter.cpp

```diff
@@ -1188,6 +1244,10 @@ void GlobalRouter::setMaxLayerForClock(const int max_layer)
   max_layer_for_clock_ = max_layer;
 }
 
+void GlobalRouter::setCriticalNetsPercentage(float critical_nets_percentage) {
+  critical_nets_percentage_ = critical_nets_percentage/100;
```

Comment:
This is confusing as the critical_nets_percentage_ sounds like it is in [0,100] not [0,1].  There isn't much advantage to doing the /100 early as this value is only used infrequently.  I would move the /100 to computeNetSlacks